### PR TITLE
Update plugin version to 21.06.0

### PR DESCRIPTION
**Pull request by pxLi**
Signed-off-by: Peixin Li <pxli@nyu.edu>

### CODE REVIEW

**Comment by pxLi** [comment]
db shim build and nightly build is currently under verification

**Comment by pxLi** [comment]
build

**Comment by pxLi** [comment]
wait for confirmation of version number

File: docs/configs.md
Comment (jlowe) [inline docs/configs.md:13]: Interesting that we're going with 21.06.0-SNAPSHOT in the plugin but 21.06-SNAPSHOT in cudf?  Should cudf be using 21.06.0-SNAPSHOT as well?

Comment (pxLi) [reply docs/configs.md:13]: yes, I am waiting for the final call for 21.06.0 vs 21.06.00 to make the change for both sides.

we used to not add the patch number for .0 release which was not good~

**Comment by pxLi** [comment]
waiting for cudfjni 21.06.0-SNAPSHOT build to run the pre-merge test

**Review by jlowe — DISMISSED** [review]

**Comment by pxLi** [comment]
build

**Comment by pxLi** [comment]
build

**Comment by pxLi** [comment]
up-merge and update spark profiling tool versioning

**Comment by pxLi** [comment]
build

**Comment by pxLi** [comment]
waiting for new cudf jar to get deployed

**Review by GaryShen2008 — DISMISSED** [review]

**Comment by pxLi** [comment]
build

**Review by GaryShen2008 — APPROVED** [review]

**Pull request merged**
